calico/felix/felix.py

```diff
@@ -49,7 +49,6 @@
 
 _log = logging.getLogger(__name__)
 
-
```

Comment:
The only changes in this file seem to de adding/deleting blank lines - intentional?